quickwit/quickwit-control-plane/src/control_plane.rs

```diff
@@ -135,6 +138,71 @@ impl Actor for ControlPlane {
     }
 }
 
+impl ControlPlane {
+    async fn delete_shards(
```

Comment:
Should we move this method to the ingest controller? So far, this is where we handle all shard mutations.